src/views/addItemView/index.tsx

```diff
@@ -0,0 +1,153 @@
+'use client';
+import TopBar from '@/components/top-bar';
+import { Box, Button, Grid, TextField, Typography } from '@mui/material';
+import { useState } from 'react';
+
+export default function AddItemView() {
+  const [itemName, setItemName] = useState('');
+  const [category, setCategory] = useState('');
+  const [highValue, setHighValue] = useState('');
+  const [lowValue, setLowValue] = useState('');
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    // Validate form fields
+    const requestData = {
+      name: itemName,
+      category: category,
+      high: Number(highValue),
+      low: Number(lowValue),
+    };
+
+    try {
+      // Send POST request to add item
+      const response = await fetch('/api/items', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(requestData),
+      });
+      // Check if response is successful
+      if (response.ok) {
+        // Handle successful response
+        alert('Item added successfully');
+        // Reset form fields
+        setItemName('');
+        setCategory('');
+        setHighValue('');
+        setLowValue('');
+      } else {
+        // Handle error response
+        console.error('Failed to add item');
+      }
+    } catch (error) {
+      console.error('Error:', error);
+    }
+  };
+
+  return (
+    <>
+      <TopBar />
```

Comment:
Go ahead and move this element into the server component `app/tiem/add/page`

Reply:
Done